util/shared/json-schema.mjs

```diff
@@ -36,6 +41,70 @@ const getAllSchemas = _ => {
   return Object.values(schemas)
 }
 
+const getExternalMarkdownPaths = obj => {
+  return objectPaths(obj)
+    .filter(x => /\/\$ref$/.test(x))
+    .map(refToPath)
+    .filter(x => /^file:/.test(getPathOr(null, x, obj)))
+}
+
+const addExternalMarkdown = obj => {
+  const paths = getExternalMarkdownPaths(obj)
+
+  paths.map(path => {
+    // grab url
+    const urn = getPathOr(null, path, obj)
+    const url = urn.indexOf("file:../") == 0 ? urn.substr("file:../".length) : urn.substr("file:".length)
+    const md = getMarkdown(url)// || "Jeremy was here..."
+
+    // drop ref
+    path.pop();
+
+    // grab field name
+    const field = path.pop()
+
+    // reassign value
+    const node = getPathOr(null, path, obj)
+    node[field] = md
+  })
+
+  return obj
+
+/*
```

Comment:
Is there a reason to keep this lying around?

Reply:
Well, it's used in a stream to add all of the markdown content to the obj passed in. if we don't return it, it wouldn't be very streamy. If we take this out, we'd have to change the `.map()` to a `.tap()` in the `getSchemaContent` method.

At the moment it's the final step in the stream, but other steps could be added in the future.

Reply:
I suppose you could make the case that it should make a copy of the object and do its operations.

Reply:
Oh... 😳

Uhm... I wanted "Jeremy was here" in the commit history?

Fixed.